ntoskrnl/kd/kdio.c

```diff
@@ -609,6 +609,14 @@ KdSendPacket(
 
     if (PacketType != PACKET_TYPE_KD_DEBUG_IO)
     {
+#ifndef KDBG
```

Comment:
Should these be `#if !DBG`?

Reply:
> Ideally it should be unconditionally compiled.

Do you mean this PR is mergeable, if I remove the `#if`s?
Or does it need more/different changes? Or...

> see https://github.com/HBelusca/reactos/tree/kdbg_tests_kdio_split/ntoskrnl/kd

That branch is dated 2023.08, so I am not sure how to usefully "compare" my (current master) PR to it.